client/src/components/Indicators.tsx

```diff
@@ -218,21 +218,14 @@ class IndicatorsWithoutI18n extends Component<IndicatorsProps, IndicatorsState>
                       <Trans>Select a Dataset:</Trans>
                     </em>{" "}
                     <br />
-                    <IndicatorsDatasetRadio
-                      id="complaints"
-                      activeId={activeVis}
-                      onChange={this.handleVisChange}
-                    />
-                    <IndicatorsDatasetRadio
-                      id="viols"
-                      activeId={activeVis}
-                      onChange={this.handleVisChange}
-                    />
-                    <IndicatorsDatasetRadio
-                      id="permits"
-                      activeId={activeVis}
-                      onChange={this.handleVisChange}
-                    />
+                    {Object.keys(INDICATORS_DATASETS).map((datasetKey, i) => (
```

Comment:
Hrmmmm I couldn't think of a better way to map over all the possible values of the `IndicatorsDatasetId` type... any suggestions?

Reply:
that's awesome, thank you @samaratrilling! I think I prefer the solution listed in there tbh